ui/pages/institutional/custody/custody.js

```diff
@@ -259,8 +259,7 @@ const CustodyPage = () => {
       if (Object.keys(connectRequestValue).length) {
         setConnectRequest(connectRequestValue);
         setCurrentJwt(
-          connectRequestValue.token ||
-            (await dispatch(mmiActions.getCustodianToken())),
+          connectRequestValue.token || dispatch(mmiActions.getCustodianToken()),
```

Comment:
A dispatch should not be required for getter function I think.

Reply:
Thanks, @jpuri. I'll address, if needed, these changes in a following PR.